src/test/java/bio/terra/workspace/db/WorkspaceDaoTest.java

```diff
@@ -137,18 +152,27 @@ public void deleteNonExistentWorkspaceFails() throws Exception {
 
   @Test
   public void duplicateWorkspaceFails() throws Exception {
-    workspaceDao.createWorkspace(workspaceId, Optional.empty(), WorkspaceStage.RAWLS_WORKSPACE);
+    Workspace workspace =
+        Workspace.builder()
+            .workspaceId(workspaceId)
+            .workspaceStage(WorkspaceStage.RAWLS_WORKSPACE)
+            .build();
```

Comment:
optional: For the tests where the exact shape of the `Workspace` isn't important (just that a `Workspace` exists), consider making a `defaultWorkspace` private variable or function to save some boilerplate and focus the lines in the test on what matters to them. As/if we add more fields to `Workspace` this becomes more helpful, whereas with 2 fields it's not as relevant. Ref: https://engdoc.corp.google.com/eng/doc/devguide/testing/unit/best-practices.md?cl=head&polyglot=java#constants